src/util/jira-utils.ts

```diff
@@ -68,6 +68,11 @@ let regexFixFeature = false;
 onFlagChange(BooleanFlags.REGEX_FIX, async () => {
 	regexFixFeature = await booleanFlag(BooleanFlags.REGEX_FIX, false);
 });
+
+let issueKeyRegexCharLimitFeature = false;
+onFlagChange(BooleanFlags.ISSUEKEY_REGEX_CHAR_LIMIT, async () => {
+	regexFixFeature = await booleanFlag(BooleanFlags.ISSUEKEY_REGEX_CHAR_LIMIT, false);
```

Comment:
I think you've assigned this to the wrong variable name :)

Reply:
oh my...